src/pages/core/guides/oracle-plugin-example.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Create a US Market Trading Experience for Your Collection!
+metaTitle: Core - Oracle Plugin Example
+description: This guide shows how to limit your collection to trade only during US market hours.
+---
+
+This developer guide leverages the new Oracle Plugin to **create an NFT collection that can only be traded during US market hours**.
+
+## Introduction
+
+### External Plugin
+
+An **External Plugin** is a plugin whose behavior is controlled by an *external* source. The core program will provide an adapter for these plugins, but developers decide the behavior by pointing this adapter to an external data source.
+
+Each External Adapter has the ability to assign lifecycle checks to Lifecycle Events, influencing the behavior of the lifecycle event taking place. This means we can assign the following checks to lifecycle events like create, transfer, update, and burn:
+- **Listen**: A “web3” webhook that alerts the plugin when a lifecycle event occurs. This is particularly useful for tracking data or performing actions.
+- **Reject**: The plugin can reject a lifecycle event.
+- **Approve**: The plugin can approve a lifecycle event.
+
+If you want to learn more about External Plugins, read more about them [here](/core/external-plugins/oracle).
```

Comment:
We should probably point to https://developers.metaplex.com/core/external-plugins/overview instead. since this is the page explaining external plugins in general